scheds/include/lib/atq.h

```diff
@@ -17,8 +17,9 @@ struct scx_atq {
 
 typedef __arena scx_atq, scx_atq_t;
 
-u64 scx_atq_create_internal(bool fifo);
-#define scx_atq_create(fifo) (scx_atq_create_internal((fifo)))
+u64 scx_atq_create_internal(bool fifo, size_t capacity);
+#define scx_atq_create(fifo) scx_atq_create_internal((fifo), SCX_ATQ_MAX_CAPACITY)
+#define scx_atq_create_size(fifo, capacity) scx_atq_create_internal((fifo), (capacity))
```

Comment:
How come we're using macros for this sort of thing instead of `__always_inline` functions?

Reply:
Just following the style of @etsal